### PR TITLE
New documentation Section and dbt core course

### PR DESCRIPTION
**Pull request by smjohnsonShipyard**
@blakeburch I tried to do the redirect thing that you mentioned. Did I do it right?

### CODE REVIEW

**Review by blakeburch — COMMENTED** [review]
General feedback:
1) I don't think this needs to block initial release, but extra screenshots about how something should look when you fill it out is always helpful. Also, highlighting the field to be clicked is helpful. I would use Zappy or similar to help with that.
2) Headers should use H2 and H3 for nesting on the table of contents. I think you used H3 throughout,.

**Comment by blakeburch** [comment]
@smjohnsonShipyard Confirming that I'm officially done here. :)

File: docs/data-packages/dbt-core/dbt-core-day-3.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+![](../.gitbook/assets/../../../.gitbook/assets/shipyard_2022_05_31_11_16_55.png)
+
+2. Hover over the blue **New** button in the top right corner of your page and click **New Fleet**.
+3. Click the gear icon on the sidebar of the Fleet Builder to open **Fleet Settings**.
+4. Under **Fleet Name**, enter `dbt Run and Download Documents`. Click on the plus sign on the Fleet Builder sidebar to begin adding Vessels.
+5. Scroll to Org Blueprints and find the Blueprint that was created in part 2 of the guide called `dbt - Execute CLI Command`. Click the Blueprint to add it to the Fleet as a Vessel.
+6. Under **Vessel Name**, enter `dbt Model Run`. Leave the **dbt CLI Command** as the default.
+
+### Generate Documentation on the dbt Run
```
[the remark this answers is not in the document]
Comment (smjohnsonShipyard) [reply docs/data-packages/dbt-core/dbt-core-day-3.md]: Solely to run them separately to show the two processes running.

File: docs/data-packages/dbt-core/dbt-core-day-3.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+#### Other Options for Documentation Export
+
+* [Azure Blob Storage](https://www.shipyardapp.com/docs/blueprint-library/azure-blob-storage/azure-blob-storage-overview/)
+* [Box](https://www.shipyardapp.com/docs/blueprint-library/box/box-overview/)
+* [Dropbox](https://www.shipyardapp.com/docs/blueprint-library/dropbox/dropbox-overview/)
+* [Email](https://www.shipyardapp.com/docs/blueprint-library/email/email-overview/)
+* [Google Cloud Storage](https://www.shipyardapp.com/docs/blueprint-library/google-cloud-storage/google-cloud-storage-overview/)
+* [Google Drive](https://www.shipyardapp.com/docs/blueprint-library/google-drive/google-drive-overview/)
+* [Slack](https://www.shipyardapp.com/docs/blueprint-library/slack/slack-overview/)
```
[the remark this answers is not in the document]
Comment (smjohnsonShipyard) [reply docs/data-packages/dbt-core/dbt-core-day-3.md]: I did not hit automation with triggers at all.

File: docs/data-packages/dbt-core/dbt-core-day-3.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (blakeburch) [reply docs/data-packages/dbt-core/dbt-core-day-3.md]: Mmk. No worries. [](edited-with-Axolo)

File: docs/data-packages/dbt-core/dbt-core-day-3.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (blakeburch) [reply docs/data-packages/dbt-core/dbt-core-day-3.md]: Mmk [](edited-with-Axolo)

File: docs/data-packages/dbt-core/dbt-core-set-up-dbt.md [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+
+![](../.gitbook/assets/../../../.gitbook/assets/shipyard_2022_05_20_11_20_42.png)
+
+
+## dbt Project File Setup
+1. Change the project name to `soccer_538`.
+2. Change the profile to `soccer_538`.
+3. Change model name to `soccer_538`.
+4. Under the soccer_538 model, add a `staging` and `marts` folder that are both materialized as views.
+5. Save your changes.
```
Comment (blakeburch) [inline docs/data-packages/dbt-core/dbt-core-set-up-dbt.md:39]: I want to confirm I'm understanding this correctly.... this all happens in the `dbt_project.yml` file? I didn't see all of these fields in the screenshot above, so I'm still a little unsure of what the correct setup is and how to confirm that I did the right thing.

**Review by blakeburch — APPROVED** [review]
Think we're good to go

**Pull request merged**